Engine/Time.h

```diff
@@ -4,30 +4,29 @@
 
 const int game_starting_year = 1168;
 
+// Number of game ticks per 30 game seconds
 #define TIME_QUANT                  128
 #define TIME_SECONDS_PER_QUANT      30
-#define TIME_UNPACK_GAME_SECONDS    (uint64_t)TIME_SECONDS_PER_QUANT / (double)TIME_QUANT
-// TIME_UNPACK_GAME_SECONDS = 0.234375
-// 30 game seconds per one time quant (128ms)   [128 * 0.234375 = 30]
-// seconds = game_time * TIME_UNPACK_GAME_SECONDS
-#define TIME_PACK_GAME_SECONDS      (uint64_t)TIME_QUANT / (double)TIME_SECONDS_PER_QUANT
-// game_time += seconds * TIME_PACK_GAME_SECONDS
+
+#define GAME_TIME_TO_SECONDS(VALUE) ((VALUE) * static_cast<uint64_t>(TIME_SECONDS_PER_QUANT) / TIME_QUANT)
+#define SECONDS_TO_GAME_TIME(VALUE) ((VALUE) * static_cast<uint64_t>(TIME_QUANT) / TIME_SECONDS_PER_QUANT)
```

Comment:
Worth adding an extra set of brackets so multiplication is done first?

Reply:
It's always done first, no need for excessive parentheses, see https://en.cppreference.com/w/cpp/language/operator_precedence